rudder-web/pom.xml

```diff
@@ -78,7 +80,7 @@ along with this program. If not, see <http://www.gnu.org/licenses/agpl.html>.
       <groupId>com.normation.rudder</groupId>
       <artifactId>rudder-core</artifactId>
       <version>${rudder-version}</version>
-    </dependency>
+     </dependency>
```

Comment:
I Think this space should not be added (not aligned with the 'dependency' below)